### ISSUE TITLE
Possible regression in accuracy of A42() in v2.0.11

### ISSUE THREAD

**Issue by judober** [JuliaMath/Roots.jl#368]
I noticed that the following code
```
f(x) = sinpi(-1/40 + x / 40) + 1 - x
z = find_zero(f, (0,2), A42())
f(z)
```
returns `0.0` in v2.0.10 but  `1.1102230246251565e-16` in v2.0.11 and v2.0.12 thus not the exact root.
The root found in v2.0.10 is `1.0`  and `0.9999999999999999` in the newer versions.

The root is correct when not using A42().

Is this to be expected?

**Comment by jverzani** [comment]
Yes, there were some changes to that algorithm to more closely match the original paper. These two are the "same" up to tolerances, but let me look to see if something can be done.

**Comment by jverzani** [comment]
Closed by #371
